requirements/extras/huggingface_requirements.txt

```diff
@@ -0,0 +1,2 @@
+accelerate
+huggingface_hub
```

Comment:
lets pin all these dependencies to avoid surprises due to auto consumption of these.

Reply:
Thanks, addressed